modules/tinymce/src/themes/silver/main/ts/ui/general/Button.ts

```diff
@@ -79,6 +79,21 @@ export const renderIconButton = (spec: IconButtonWrapper, action: (comp: AlloyCo
   return AlloyButton.sketch(iconButtonSpec);
 };
 
+const calculateClassesFromButtonType = (buttonType: 'primary' | 'secondary' | 'toolbar') => {
+  switch (buttonType) {
+    case 'primary':
+      return [ 'tox-button' ];
+      break;
+    case 'toolbar':
+      return [ 'tox-tbtn' ];
+      break;
+    case 'secondary':
+    default:
```

Comment:
Do we need a fall-though here to default? Or is typescript protecting us against passing in something completely random here. Boulder should have validated any user input before this.

Reply:
It's not strictly necessary, but I figure it's a nice protection to have in case of future potential refactorings.